menas/src/test/resources/application.properties

```diff
@@ -30,21 +30,23 @@ za.co.absa.enceladus.menas.auth.ldap.search.filter=(&(samAccountName={0}))
 # Define how menas authenticates to Hadoop. Supported options are:
 #  "default" -> will use the default authentication or kerberos ticket from the system
 #  "krb5" -> menas will use specified kerberos configurations to authenticate via kerberos using specified username & keytab
-za.co.absa.enceladus.menas.hadoop.auth.method=krb5
+za.co.absa.enceladus.menas.hadoop.auth.method=default
 za.co.absa.enceladus.menas.hadoop.auth.krb5.debug=true
 za.co.absa.enceladus.menas.hadoop.auth.krb5.realm=EXAMPLE.COM
 za.co.absa.enceladus.menas.hadoop.auth.krb5.kdc=localhost
 za.co.absa.enceladus.menas.hadoop.auth.krb5.username=hdfs@EXAMPLE.COM
 za.co.absa.enceladus.menas.hadoop.auth.krb5.keytab=hdfs.keytab
 
 za.co.absa.enceladus.menas.auth.inmemory.user=user
-za.co.absa.enceladus.menas.auth.inmemory.password=changeme
+za.co.absa.enceladus.menas.auth.inmemory.password=chang< )eme
```

Comment:
typo ?

Reply:
No, I wanted to have the tests run with a different password than what is default in the `template` and to include odd characters to be sure they are handled properly by the rest client.